lib/view_model/active_record/update_context.rb

```diff
@@ -3,18 +3,31 @@
 # the mechanism by which it is applied to models.
 class ViewModel::ActiveRecord
   class UpdateContext
+    attr_reader :referenced_update_data # TODO: eeeep
+
     ReleaseEntry = Struct.new(:viewmodel, :association_data) do
```

Comment:
Use of this class has escaped this file. Do you think that's enough reason to give it its own file?